raiden/network/discovery.py

```diff
@@ -45,7 +45,8 @@ class ContractDiscovery(Discovery):
     """On chain smart contract raiden node discovery: allows to register endpoints (host, port) for
     your ethereum-/raiden-address and looking up endpoints for other ethereum-/raiden-addressess.
     """
-    def __init__(self, blockchainservice, discovery_contract_address, poll_timeout=DEFAULT_POLL_TIMEOUT):
+    def __init__(self, blockchainservice, discovery_contract_address,
```

Comment:
Humm, the suggestion doesnt fully comply with the Python's coding standard [PEP8](https://www.python.org/dev/peps/pep-0008/#indentation):

> When using a hanging indent the following should be considered; there should be no arguments on the first line and *further indentation should be used* to clearly distinguish itself as a continuation line.

```python
# More indentation included to distinguish this from the rest.
def long_function_name(
        var_one,
        var_two,
        var_three,
        var_four):
    print(var_one)
```

Pylint is the tool erroring in this (the styling tools are not), so I will have a look on how to configure this (I have no preference among the hanging indent options).